File: app/src/main/java/com/example/rongyi/baseandroidmodel/MyApplication.java
package com.example.rongyi.baseandroidmodel;

import android.app.Application;

import com.orhanobut.logger.AndroidLogAdapter;
import com.orhanobut.logger.Logger;

/**
 * Created by rongyi on 2017/6/22.
 */

public class MyApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();

        Logger.addLogAdapter(new AndroidLogAdapter() {
            @Override
            public boolean isLoggable(int priority, String tag) {
                return BuildConfig.SHOW_LOG;
            }
        });

    }

}

File: app/src/main/java/com/example/rongyi/baseandroidmodel/MyApplication.kt
package com.example.rongyi.baseandroidmodel

import android.app.Application
import com.safframework.log.L

/**
 * Created by rongyi on 2017/6/22.
 */

class MyApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        if (BuildConfig.SHOW_LOG) {
            L.logLevel = L.LogLevel.DEBUG
        } else {
            L.logLevel = L.LogLevel.INFO
        }
    }


}
